Makefile

```diff
@@ -16,7 +16,7 @@ zig.wasm: main.zig
 	zig build-exe -target wasm32-wasi-musl -O ReleaseFast $< -femit-bin=$@
 
 rust.wasm: main.rs
-	rustc $< -O --target wasm32-wasi -o $@
+	rustc $< -O -C strip=debuginfo  --target wasm32-wasi -o $@
```

Comment:
please, compare `strip=debuginfo` to `strip=symbols`
`symbols stripped`: 72326 (71K)

Reply:
`-C strip=debuginfo -C strip=symbols`: 72326 rust.wasm